amlb/runners/docker.py

```diff
@@ -51,9 +51,22 @@ def _start_container(self, script_params=""):
             touch(d, as_dir=True)
         script_extra_params = ""
         inst_name = self.sid
+
+        if ' -f ' in script_params:
+             inst_name = inst_name + '_f' + script_params.split(' -f ')[1].split()[0]
+        elif ' --fold ' in script_params:
+            inst_name = inst_name + '_f' + script_params.split(' --fold ')[1].split()[0]
```

Comment:
how do you run your multiple docker instances?
1. separately?
2. using `--parallel=2` for example?

if (1), then you can specify the sid (and therefore the instance name) in the command line using `--session=...`

if (2), then we have to generate the `inst_name` properly: 
- `script_params` is generated internally in `container.py` and there can't be any `--fold` param, it's always `-f`.
- in parallel mode, we can also have same fold, but different tasks.
- I don't like the idea of parsing a command that is generated by the caller, this is pretty dirty.

If possible, I'd prefer for you to use first approach for now (using `--session`) and I'll think about a consistent way to support parallel jobs with containers.